pkg/scheduler/apis/config/types.go

```diff
@@ -94,6 +95,27 @@ type KubeSchedulerConfiguration struct {
 	// failover with the benefit of lower memory overhead while waiting to become leader.
 	// Defaults to false.
 	DelayCacheUntilActive bool
+
+	// Metric defines the configuration of metrics.
+	Metric KubeSchedulerMetricConfiguration
+}
+
+type KubeSchedulerMetricConfiguration struct {
+	// Percentage of plugin metrics to sample.
+	// A value of 100 means all metrics are recorded; lower values enable probabilistic sampling.
+	PluginMetricsSamplePercent int
+
+	// Buffer size for the MetricsAsyncRecorder.
+	// Incoming metrics are dropped when the buffer is full.
+	// Increasing the buffer size (with the trade-off of higher memory consumption) may be necessary
+	// to avoid metric events being dropped due to buffer overflow, particularly in large clusters.
+	MetricsAsyncRecorderBufferSize int
+
+	// Flush interval for the MetricsAsyncRecorder.
+	// Defines how frequently buffered metrics are pushed to the underlying sink.
+	// Reducing the buffer flush interval (with the trade-off of higher CPU consumption) may be necessary
+	// to avoid metric events being dropped due to buffer overflow, particularly in large clusters.
+	MetricsAsyncRecorderFlushInterval metav1.Duration
```

Comment:
Would you mind validating the values? For example, could you please ensure that PluginMetricsSamplePercent is within the range of 0 to 100?